cc-webgraph-statistics/README.md

```diff
@@ -0,0 +1,239 @@
+
+
+Once this is defined, we may generate offset files for the graph and the transpose graph:
+```
+$WG it.unimi.dsi.webgraph.BVGraph -O -L cc-main-2020-feb-mar-may-domain
+$WG it.unimi.dsi.webgraph.BVGraph -O -L cc-main-2020-feb-mar-may-domain-t
+```
+
+Offset files are used to determine where each successor list is stored (successor lists are what makes up the graph file, each node is represented by a successor list). More information on the [offset files](http://webgraph.di.unimi.it/docs/it/unimi/dsi/webgraph/BVGraph.html) section.
+
+We can then generate the files for connected components:
+
+```
+$WG it.unimi.dsi.webgraph.algo.ConnectedComponents -m --renumber --sizes -t cc-main-2020-feb-mar-may-domain-t cc-main-2020-feb-mar-may-domain
+$WG it.unimi.dsi.webgraph.algo.StronglyConnectedComponents --renumber --sizes cc-main-2020-feb-mar-may-domain
+```
+which will produce `*.wcc`, `*.wccsizes` which contains information about weakly connected components, and similarly the files `*.scc` and `*.sccsizes` contain information about strongly connected components. More info on the [weakly connected components](http://webgraph.di.unimi.it/docs/it/unimi/dsi/webgraph/algo/ConnectedComponents.html) and the [strongly connected components](http://webgraph.di.unimi.it/docs/it/unimi/dsi/webgraph/algo/StronglyConnectedComponents.html) classes, and about the concept [itself](http://pages.di.unipi.it/marino/pythonads/Graphs/StronglyConnectedComponents.html#:~:text=To%20transform%20the%20World%20Wide,connecting%20one%20vertex%20to%20another.&text=One%20graph%20algorithm%20that%20can,connected%20components%20algorithm%20(SCC).).
+
+
+Now finally, we may generate some statistics with the Stats class of WebGraph by running:
+```
+$WG it.unimi.dsi.webgraph.Stats --save-degrees cc-main-2020-feb-mar-may-domain
+```
+The --save-degrees flag generates `*.outdegrees` and `*.indegrees`  files which contain number of degrees per node (so the first line of the outdegrees file contains the number of outdegrees that node of id 0). `*.outdegree` and `*.indegree` files will be generated which contains frequency distribution of degrees (so line 0 of .indegree contains the number of nodes with 0 incoming edges). Also, if connected component files are present, Stats will genenerate some information about the connected components. More information about the [Stats class](http://webgraph.di.unimi.it/docs/it/unimi/dsi/webgraph/Stats.html).
+
+You are now ready to run the [notebook](https://github.com/commoncrawl/cc-notebooks/blob/statistics/cc-statistics/topology_stats.ipynb)! Simply load it and take a look. Double check any variables with pathing.
```

Comment:
the link to the notebook should be a relative link (just `topology_stats.ipynb`)